### PR TITLE
Apotechary chem dispenser nerf.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Made the apothecary chem dispenser into its own subtype with reduced reagents availability.

## Why It's Good For The Game
I was and am fairly disappointed about the apotechary chem dispenser. I understood the intentions of making doctors not rely on chemistry for more medicines, but in reality it was a powercrepe, they already had plenty of stuff to deal with the average round accidents.

## Changelog
:cl:
balance: nerfed the apotechary chem dispenser.
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Please place this in Rnd basic if you can, I can see people /wanting/ to make this tbh, and it would be good for consistency were basic is cheap and bio level 1 is normal

**Comment by YakumoChen** [comment]
there's already a spare dispenser board in tech storage and dispensers are pretty early in the tech tree if you really need it.

**Comment by Poojawa** [comment]
tbh it was honestly just because Chemists were useless at the time of being added. If they've improved it's fine to remove entirely.

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -38177,7 +38177,7 @@
 /obj/machinery/light/small{
 	dir = 8
 	},
-/obj/machinery/chem_dispenser,
+/obj/machinery/chem_dispenser/apotechary,
```
Comment (Ghommie) [inline _maps/map_files/BoxStation/BoxStation.dmm:38180]:
```suggestion
/obj/machinery/chem_dispenser/apothecary,
```

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -125236,7 +125236,7 @@
 /obj/effect/turf_decal/tile/neutral{
 	dir = 8
 	},
-/obj/machinery/chem_dispenser,
+/obj/machinery/chem_dispenser/apotechary,
```
Comment (Ghommie) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:125239]:
```suggestion
/obj/machinery/chem_dispenser/apothecary,
```

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 3]
```diff
@@ -38320,7 +38320,7 @@
 /obj/machinery/light/small{
 	dir = 8
 	},
-/obj/machinery/chem_dispenser,
+/obj/machinery/chem_dispenser/apotechary,
```
Comment (Ghommie) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:38323]:
```suggestion
/obj/machinery/chem_dispenser/apothecary,
```

File: _maps/map_files/LambdaStation/lambda.dmm [hunk 4]
```diff
@@ -38260,12 +38260,10 @@
 /turf/open/floor/plasteel/white,
 /area/medical/chemistry)
 "btD" = (
-/obj/machinery/chem_dispenser{
-	layer = 2.7
-	},
 /obj/effect/turf_decal/trimline/blue/filled/line{
 	dir = 1
 	},
+/obj/machinery/chem_dispenser/apotechary,
```
Comment (Ghommie) [inline _maps/map_files/LambdaStation/lambda.dmm:38266]:
```suggestion
/obj/machinery/chem_dispenser/apothecary,
```

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 5]
```diff
@@ -63926,9 +63926,7 @@
 /obj/effect/turf_decal/tile/blue{
 	dir = 8
 	},
-/obj/machinery/chem_dispenser{
-	layer = 2.7
-	},
+/obj/machinery/chem_dispenser/apotechary,
```
Comment (Ghommie) [inline _maps/map_files/MetaStation/MetaStation.dmm:63929]:
```suggestion
/obj/machinery/chem_dispenser/apothecary,
```

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 6]
```diff
@@ -692,6 +692,16 @@
 	def_components = list(/obj/item/stock_parts/cell = /obj/item/stock_parts/cell/high)
 	needs_anchored = FALSE
 
+/obj/item/circuitboard/machine/chem_dispenser/apotechary
+	name = "Apotechary Chem Dispenser (Machine Board)"
```
Comment (Ghommie) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]:
```suggestion
/obj/item/circuitboard/machine/chem_dispenser/apothecary
	name = "Apothecary Chem Dispenser (Machine Board)"
	build_path = /obj/machinery/chem_dispenser/apothecary
```

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 7]
```diff
@@ -720,3 +720,58 @@
 	component_parts += new /obj/item/stack/sheet/glass(null)
 	component_parts += new /obj/item/stock_parts/cell/bluespace(null)
 	RefreshParts()
+
+///An unique, less efficient model found in the medbay apotechary room.
+/obj/machinery/chem_dispenser/apotechary
+	name = "apotechary chem dispenser"
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]:
```suggestion
/obj/machinery/chem_dispenser/apothecary
	name = "apothecary chem dispenser"
```

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 8]
```diff
@@ -720,3 +720,58 @@
...
+	desc = "A cheaper chem dispenser meant for small scale medicine production."
+	icon_state = "minidispenser"
+	working_state = "minidispenser_working"
+	nopower_state = "minidispenser_nopower"
+	circuit = /obj/item/circuitboard/machine/chem_dispenser/apotechary
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]:
```suggestion
	circuit = /obj/item/circuitboard/machine/chem_dispenser/apothecary
```

**Review by Ghommie — COMMENTED** [review]
Spelling

**Comment by Ghommie** [comment]
Ugh, somehow the code suggestions tool doesn't work right now. That's stressing.

**Pull request merged**